features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -111,6 +114,85 @@ export default function Environments() {
             .finally(() => setUpdating(false));
     }
 
+    /**
+     *
+     * Renders the Micro gateway information
+     */
+    function MicroGateway() {
```

Comment:
Since this is a separate component shall we move it to a different file and remove the if condition, then we can do a conditional rending in line 269 like
`{!isWebsocket &&  <MicroGateway />`

Reply:
Done